test/org/opensolaris/opengrok/analysis/AnalyzerGuruTest.java

```diff
@@ -244,6 +242,15 @@ public void getAnalyzerFactoryClass() {
     }
 
     @Test
+    public void shouldNotThrowGettingCsprojOpening() throws IOException {
+        InputStream res = getClass().getClassLoader().getResourceAsStream(
+            "org/opensolaris/opengrok/analysis/a.csproj");
```

Comment:
I wonder if we should stick the test files under subdirectory or elsewhere. There does not seem to be a convention for tests yet.

Reply:
I thought about that — but a lower-cased file name does at least show it at the bottom of the test/org folder in NetBeans.

Reply:
Yeah, no big deal. We should probably rethink and change this for all tests.